server/middleware/workstack.js

```diff
@@ -66,14 +66,26 @@ function workstackApiResponseMiddleware(req, res) {
     res.json(res.locals.workstack);
 }
 
+const updateCaseDataMoveTeamRequest = async (req, res, [endpoint, body, headers]) => {
+    const logger = getLogger(req.requestId);
+    try {
+        await workflowService.put(endpoint, body, headers);
+        return;
+    } catch (error) {
+        logger.error('MOVE_TEAM_FAILED', { endpoint, body, status: error.response.status });
+        res.locals.notification = 'Failed to update all cases data with new team.';
```

Comment:
I think this copy needs work. If it's used as part of the notification element, we'll only have 1000ms to read and understand what's going on, so it needs to be brief.

(I would rather it threw a real 500 page, but we've got to work with what we've got, I guess)

Reply:
Yeah we don't actually use these `res.locals.notification` anywhere. I'm happy to keep it in for now and open a ticket to review. As we should ideally be showing this error with a 500 page as you say.

Reply:
HOCS-2917 has been created for this, resolving for now.